packages/pluggableWidgets/datagrid-dropdown-filter-web/src/utils/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { RefObject, useEffect } from "react";
+
+export function useOnClickOutside(ref: RefObject<HTMLUListElement>, handler: () => void): void {
```

Comment:
This is a useful hook! Could this be extracted to a common package? Didn't Wesley create something similar for Popup?

Reply:
I think maybe move to our internal stuff like piw-utils instead. In commons we dont provide util functions related to dom..

Reply:
Diego will take care of this. So i am skipping this to not face with merge conflicts

Reply:
Yap, my build is failing, so I need Mehmets fix, in my branch, then i will change his widget :)

Reply:
👍